jenkins/manifest.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 name: Jenkins
-version: 2.289.3
+version: 2.414.1
```

Comment:
There is something wrong on the GitHub diff...
The current version on master is `2.401.3` as per https://github.com/civo/kubernetes-marketplace/blob/ce5f42dc6e5a1c095e4ee2a587bc75d5a4079ef1/jenkins/manifest.yaml#L3

So it will bump from `2.401.3` to `2.414.1`